blueprints/app/files/tests/test-helper.js

```diff
@@ -10,3 +11,22 @@ document.write('<div id="ember-testing-container"><div id="ember-testing"></div>
 QUnit.config.urlConfig.push({ id: 'nocontainer', label: 'Hide container' });
 var containerVisibility = QUnit.urlParams.nocontainer ? 'hidden' : 'visible';
 document.getElementById('ember-testing-container').style.visibility = containerVisibility;
+
+QUnit.config.urlConfig.push({ id: 'nojshint', label: 'Disable JSHint'});
+
+if (QUnit.notifications) {
+  QUnit.notifications({
+    icons: {
+      passed: '/assets/passed.png',
+      failed: '/assets/failed.png'
+    }
+  });
+}
+
+$(document).ready(function(){
```

Comment:
I don't think this file should exist in people's apps at all. If we're going to extract some, let's extract all.